__tasks__/build-riffraff-artifact.sh

```diff
@@ -17,14 +17,18 @@ if [[ -z $BRANCH_NAME ]]; then
 fi
 
 TARGET_ROOT=target/guui
+ASSETS_ROOT=target/frontend-static
 
 zip -r guui.zip * -x node_modules/**\*
 [ -d target ] && rm -rf target
+mkdir -p $ASSETS_ROOT/moon-javascripts
```

Comment:
i feel like we're going to need to settle on one name soon! `moon` in repo called `guui`...